libimgui-render-metal/imgui/buildfile

```diff
@@ -1,7 +1,7 @@
 intf_libs = # Interface dependencies.
 import intf_libs += libimgui%lib{imgui}
 
-platform_libs = -framework AppKit -framework ModelIO -framework Metal -framework MetalKit
+platform_libs = -framework AppKit -framework ModelIO -framework Metal -framework MetalKit -framework GameController -framework QuartzCore
```

Comment:
Same here: What's the reason behind the additional flags in the non-docking package?